src/MoneyParserTrait.php

```diff
@@ -37,7 +37,11 @@ public static function parse($value, $currency = null, $bitCointDigits = 2)
 
         $currency = static::parseCurrency($currency);
 
-        if (is_int($value) || filter_var($value, FILTER_VALIDATE_INT) !== false) {
+        if (is_int($value)) {
+            return new Money($value * 100, $currency);
```

Comment:
We cannot change the value passed

Reply:
But in order to have it casted to what Money expects, we have to? At least as long as the documentation  in the readme is considered to be the intended behaviour?